tools/create_release.py

```diff
@@ -151,6 +152,14 @@ def upload(self, path: Path, mimetype: str):
         response = requests.post(self.upload_url, headers=headers, params=params, data=path.read_bytes())
         response.raise_for_status()
 
+    def create_source_fallback(self):
+        response = requests.get(self.wrap_section['source_url'])
+        response.raise_for_status()
+        filename = Path(self.wrap_section['source_filename'])
+        filename.write_bytes(response.content)
+        self.upload(filename, 'application/zip')
```

Comment:
What is the content type even used for? Seems like a pointless aspect of GitHub 's API. They should not and probably do not care whether it's a zip or a tarball. So the fact that we get this wrong probably does not matter.

Reply:
I could use `mimetypes.guess_type()` but I don't think it matters.